.github/dependabot.yml

```diff
@@ -5,17 +5,6 @@
 
 version: 2
 updates:
-  - package-ecosystem: "pip"
```

Comment:
I'm not familiar with the dependency management for `uv`, but how does it work for `uv`? I cannot see any references about keeping the dependencies up to date.